### PR TITLE
Remove good first issue labels when complete or pr available

### PR DESCRIPTION
**Pull request by TheLimeGlass**
### Description
Starting the automation of some of our standards for GitHub issues. With more and more members being added to SkriptLang, it's best to automate some of our issue labeling standards with automation rather than explaining the standards each time in Discord to new members.

This pull creates a folder with the naming of the actions file being the event related to the action. This pull acts more of an initiative to start creating some of these automations.

To start off the automation, this will remove the "good first issue" label when complete or pr available is added. This makes sense as people using the "good first issue" are looking for easy issues, but if it's already done, it shouldn't have the tag anymore.

### CODE REVIEW

**Review by Moderocky — APPROVED** [review]
Looks like a good idea

**Review by AyhamAl-Ali — APPROVED** [review]

File: .github/workflows/github-issues/issues-labeled.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+name: When labels are modified, run actions.
```
Comment (Pikachu920) [inline .github/workflows/github-issues/issues-labeled.yml:1]: LGTM except can we get a accurate name on this (the name of the PR would probably work great)?

**Review by Pikachu920 — CHANGES_REQUESTED** [review]

File: .github/workflows/github-issues/issues-labeled.yml (hunk 1 above)
Comment (TheLimeGlass) [reply .github/workflows/github-issues/issues-labeled.yml:1]: It's setup to be modified for other label changes

Comment (AyhamAl-Ali) [reply .github/workflows/github-issues/issues-labeled.yml:1]: Isn't it better for other label actions to be splitted across multiple files or be all put in one file?

Comment (TheLimeGlass) [reply .github/workflows/github-issues/issues-labeled.yml:1]: I was making the files by event type, hence the file naming scheme

**Review by sovdeeth — APPROVED** [review]

**Pull request merged**